src/control/lib/netdetect/netdetect.go

```diff
@@ -177,72 +115,25 @@ type DeviceScan struct {
 	targetDevice         string
 }
 
-type logger interface {
-	Debug(string)
-	Debugf(string, ...interface{})
-}
```

Comment:
For the record, the main reason I asked for this interface was to make sure that the library could only do debug logging, not info/error logging.

Reply:
Okay, I can put it back in.